packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1355,54 +1357,60 @@ export default class SchedulerTask {
                     `Uploading dashboard with ${chartUuids.length} charts to Google Sheets`,
                 );
                 // We want to process all charts in sequence, so we don't load all chart results in memory
-                chartUuids.reduce(async (promise, chartUuid) => {
-                    await promise;
-                    const chart =
-                        await this.schedulerService.savedChartModel.get(
-                            chartUuid,
-                        );
-                    const { rows } =
-                        await this.projectService.getResultsForChart(
+                chartUuids
```

Comment:
Since the entire operation is a promise, I've just added a .catch statement and a logger. You can scroll away!

Reply:
Could this instead be replaced with an `await`/`try/catch` block?

Out of scope for this, but I'd probably also rewrite this as a `for/of` block to make the fact that it runs in a sequence easier to follow.

Not treating this as a blocker regardless.

Reply:
great suggestion! I'll check that